scipy/special/_logsumexp.py

```diff
@@ -93,43 +96,52 @@ def logsumexp(a, axis=None, b=None, keepdims=False, return_sign=False):
     1.6094379124341005, 1.6094379124341005
 
     """
-    a = _asarray_validated(a, check_finite=False)
+    xp = array_namespace(a, b)
+    if is_jax(xp):
+        return xp.special.logsumexp(a, axis=axis, b=b, keepdims=keepdims,
+                                    return_sign=return_sign)
```

Comment:
Ugh, I think I see now from `scipy_namespace_for` that I have to do:
```suggestion
        import jax
        return jax.scipy.special.logsumexp(a, axis=axis, b=b, keepdims=keepdims,
                                           return_sign=return_sign)
```
Do we want to go down this route or skip JAX in tests to remind us to come back when we have a utility (that mutates if possible and copies otherwise)?

Reply:
Either option seems fine. Skipping tests and circling back to them at a later point seems the most pragmatic - right now it is useful to cover a lot of ground quickly and discover any problems, while everything is still experimental. We will be in a better position to address test failures and polish things up at a later point.

Reply:
Skipping for now.